source/agora/node/Ledger.d

```diff
@@ -252,7 +252,8 @@ unittest
                 assert(ledger.acceptTransaction(tx));
             });
 
-        last_txs = txes;
+        // keep track of last tx's to chain them to
+        last_txs = txes[$ - Block.TxsInBlock .. $];;
```

Comment:
Shouldn't it be:
```suggestion
        last_txs = txes[$ - (Block.TxsInBlock * count) .. $];
```

Reply:
I just want the last 8 transactions, regardless of how many are there before that.

Reply:
Ah right: https://github.com/bpfkorea/agora/blob/f5bed737df2d19a1e83b6bf2ea4b5121f27d7903/source/agora/common/Transaction.d#L348

Maybe the function should reflect that

Reply:
```suggestion
        last_txs = txes[$ - Block.TxsInBlock .. $];
```

Reply:
> Maybe the function should reflect that

You mean to add docs? Sure.